README.md

```diff
@@ -34,7 +34,7 @@ const cid = CID.create(1, json.code, hash)
 ### Creating Blocks
 
 ```js
-import Block from 'multiformats/block'
+import { Block } from 'multiformats/block'
```

Comment:
It should be `import * as Block from 'multiformats/block'` as the `encode`, `decode` and `create` factories should be the primary interface, as detailed in this example. The constructor is less helpful by itself.